packages/launch/src/focused-launch/summary/style.scss

```diff
@@ -193,10 +193,6 @@ $commentary-column-gap-wide: 100px;
 
 	.focused-launch-summary__launch-button {
 		flex: 1;
```

Comment:
Can we investigate a bit more into the optimal values for `flex-grow` and `flex-shrink`? I assume we could have grow: 1 and shrink: 0.

Also, we should probably add a margin-right on > small screens to this button.

Finally, we should test the edge case where the button's text is very very long, and see how the flex layout behaves (is the help text to the right pushed smaller? Or what else?)

Reply:
> Can we investigate a bit more into the optimal values for flex-grow and flex-shrink? I assume we could have grow: 1 and shrink: 0.

I think `flex-grow` is enough to contain verbose copy inside the button's screen estate. I've pushed a new commit for that. 

> Also, we should probably add a margin-right on > small screens to this button.

No, we don't since we're already applying `justify-content: space-between;` to viewports `> small`.

Reply:
> Finally, we should test the edge case where the button's text is very very long, and see how the flex layout behaves (is the help text to the right pushed smaller? Or what else?)

In the current situation, the launch button will push the help text to the right, consuming all the space (see screenshot). 

<img width="750" alt="Screenshot 2021-02-09 at 16 19 44" src="https://user-images.githubusercontent.com/12104589/107384864-a783e180-6af2-11eb-888d-fbb3ed664759.png">

What I propose in situations where the button's text is very, very long, is to stack the help text on top of the button as we do for mobile viewports. What do you think @ollierozdarz , @ciampo?

Reply:
> No, we don't since we're already applying justify-content: space-between; to viewports > small.

The `margin-right` on the button was to guarantee some white space between the button and the text when the button gets very long — but I just checked the code, and it looks like the flexbox parent already has a `gap: 20px` rule. So, no action needed there!

> What I propose in situations where the button's text is very, very long, is to stack the help text on top of the button as we do for mobile viewports.

I personally like that idea, since it basically aligns to what we're already doing on mobile (probably no need to wait for Ollie's blessing either!)

I toyed with that a little, and came up with these changes:

```diff
--- a/packages/launch/src/focused-launch/summary/style.scss
+++ b/packages/launch/src/focused-launch/summary/style.scss
@@ -152,19 +152,22 @@ $commentary-column-gap-wide: 100px;
                display: flex;
                justify-content: space-between;
                align-items: center;
+               gap: 20px;
+               flex-wrap: wrap-reverse;
        }

        @include break-medium {
                flex-direction: column-reverse;
                align-items: flex-start;
-               gap: 20px;
                width: calc( 100% - #{$commentary-column-width-medium + $commentary-column-gap-small} );
+               flex-wrap: nowrap;
        }

        @include break-large {
                flex-direction: row;
                align-items: center;
                width: calc( 100% - #{$commentary-column-width-large + $commentary-column-gap-large} );
+               flex-wrap: wrap-reverse;
        }

        @include break-wide {
@@ -178,6 +181,7 @@ $commentary-column-gap-wide: 100px;

        @include break-small {
                margin-bottom: 0;
+               flex-shrink: 0;
        }
```

This is how it'd look:

https://user-images.githubusercontent.com/1083581/107394830-8e802e00-6afc-11eb-81cf-2c32c3d1f894.mp4

Reply:
Thanks for the suggestions! I've implemented it and it works like a charm.